pyproject.toml

```diff
@@ -182,6 +182,7 @@ exclude = [
   "temporalio/bridge/metric.py",
   "temporalio/bridge/runtime.py",
   "temporalio/bridge/testing.py",
+  "temporalio/envconfig.py",
```

Comment:
Why is this excluded?

Reply:
`envconfig.py` encounters the same error in the CI as the one mentioned in a comment a few lines above:
```
  # TODO: these pass locally but fail in CI with
  # error: Import "temporalio.bridge.temporal_sdk_bridge" could not be resolved
  ```